source/features/show-names.tsx

```diff
@@ -18,56 +18,60 @@ function dropExtraCopy(element: HTMLAnchorElement): void {
 	}
 }
 
-// The selector observer calls this function several times, but we want to batch them into a single GraphQL API call
-async function updateLink(batchedUsernameElements: HTMLAnchorElement[]): Promise<void> {
-	const myUsername = getUsername();
-	batchedUsernameElements = batchedUsernameElements.filter(({textContent: name}) => name !== myUsername && name !== 'ghost');
-	const usernames = new Set(batchedUsernameElements.map(element => element.textContent));
+function appendName(element: HTMLAnchorElement, fullName: string): void {
+	// If it's a regular comment author, add it outside <strong> otherwise it's something like "User added some commits"
+	const {parentElement} = element;
+	const insertionPoint = parentElement!.tagName === 'STRONG' ? parentElement! : element;
+
+	insertionPoint.after(
+		' ',
+		<span className="color-fg-muted css-truncate d-inline-block">
+			(<bdo className="css-truncate-target" style={{maxWidth: '200px'}}>{fullName}</bdo>)
+		</span>,
+		' ',
+	);
+}
 
-	if (usernames.size === 0) {
+async function updateLink(found: HTMLAnchorElement[]): Promise<void> {
+	// eslint-disable-next-line no-use-extend-native/no-use-extend-native -- Sigh
+	const users = Map.groupBy(found, element => element.textContent.trim());
```

Comment:
New: `.trim()`. It hasn't been an issue so far, but…
New: Read `textContent` once, to avoid race conditions